Monika After Story/game/script-ch30.rpy

```diff
@@ -520,26 +520,32 @@ label ch30_autoload:
     #Run actions for any events that are based on the clock
     $ evhand.event_database=Event.checkCalendar(evhand.event_database)
 
-    #Skip all greetings if you closed the game on Monika
-    if persistent.closed_self:
-        #pick a random greeting
-        if is_monika_in_room:
-            if persistent.current_monikatopic != "i_greeting_monikaroom":
-                $ pushEvent("i_greeting_monikaroom")
-        else:
-            $pushEvent(renpy.random.choice(greetings_list))
+    #if the player told Monika that they are sick, jump to this greeting.
+    if persistent.sick:
+        $ pushEvent("greeting_sick")
```

Comment:
You're not supposed to get a greeting if you didn't say good bye to monika, so this should be in the `closed_self` flow, specifically as an elif for the `is_monika_in_room`.